CMakeLists.txt

```diff
@@ -889,9 +889,9 @@ if (USE_TENSORRT)
 	set(TENSORRT_LIBS protobuf nvinfer nvparsers nvinfer_plugin nvonnxparser )
 
   else()
-    set(TENSORRT_LIB_DIR ${CMAKE_BINARY_DIR}/tensorrt-oss/src/tensorrt-oss-build ${TENSORRT_LIB_DIR})
+    set(TENSORRT_LIB_DIR ${CMAKE_BINARY_DIR}/tensorrt-oss/src/tensorrt-oss-build ${CMAKE_BINARY_DIR}/tensorrt-oss/bin/ ${TENSORRT_LIB_DIR})
```

Comment:
I think ${CMAKE_BINARY_DIR}/tensorrt-oss/src/tensorrt-oss-build can be dropped as everything should be in ${CMAKE_BINARY_DIR}/tensorrt-oss/bin/  because of -DTRT_OUT_DIR=${CMAKE_BINARY_DIR}/tensorrt-oss/bin set some lines below

Reply:
Good intuition! the build goes through when removing this path, I've updated the commit.